lib/sinatra/show_exceptions.rb

```diff
@@ -24,23 +24,22 @@ def call(env)
 
       if prefers_plain_text?(env)
         content_type = "text/plain"
-        exception_string = dump_exception(e)
+        exception = dump_exception(e)
       else
         content_type = "text/html"
-        exception_string = pretty(env, e)
+        exception = pretty(env, e)
       end
 
       env["rack.errors"] = errors
-
-      # Post 893a2c50 in rack/rack, the #pretty method above, implemented in
-      # Rack::ShowExceptions, returns a String instead of an array.
```

Comment:
@kgrz yeah, it is in the full commit message: https://github.com/lucasdavila/sinatra/commit/49751223442a6ae4d62a5505f6b6a2c297df470e

But explaining better: You don't need to comment the code to register the reason why it changed (in this case because rack changed the return type of the method pretty) like you did in the last commit.

You just need to change the code, and register the reason for the change in the commit's message.

Also, one more reason to remove the comment is that the code now make clear that the expected return is an array, but that it could be a string (and that it will be wrapped in an array in this case).

So I don't see a reason to comment the code.